src/bundle/DependencyInjection/Configuration/Parser/Security.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUiBundle\DependencyInjection\Configuration\Parser;
+
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\AbstractParser;
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\SiteAccessAware\ContextualizerInterface;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+/**
+ * Configuration parser for location ids declaration.
+ *
+ * Example configuration:
+ * ```yaml
+ * ezpublish:
+ *   system:
+ *      default: # configuration per siteaccess or siteaccess group
+ *          security:
+ *              token_ttl: 10
+ * ```
+ */
+class Security extends AbstractParser
+{
+    /**
+     * Adds semantic configuration definition.
+     *
+     * @param NodeBuilder $nodeBuilder Node just under ezpublish.system.<siteaccess>
+     */
+    public function addSemanticConfig(NodeBuilder $nodeBuilder)
+    {
+        $nodeBuilder
+            ->arrayNode('security')
+                ->info('System security configuration')
+                ->children()
+                    ->scalarNode('token_ttl')->isRequired()->end()
```

Comment:
If this supposed to be `DateInterval` period, then naming is not good. Time To Live indicates usually seconds.

It rather should be `token_interval_spec` based on [`DateInterval::__construct` arguments doc](http://php.net/manual/pl/dateinterval.construct.php)

Here some description (provided by `info()` AFAIR) could also link to that mentioned doc on interval specification.